optimizely/config/polling_manager.go

```diff
@@ -64,29 +64,29 @@ func SetDefaultRequester(sdkKey string) OptionFunc {
 	}
 }
 
-// SetRequester is an optional function, sets a passed requester
-func SetRequester(requester utils.Requester) OptionFunc {
+// Requester is an optional function, sets a passed requester
+func Requester(requester utils.Requester) OptionFunc {
 	return func(p *PollingProjectConfigManager) {
 		p.requester = requester
 	}
 }
 
-// SetPollingInterval is an optional function, sets a passed polling interval
-func SetPollingInterval(interval time.Duration) OptionFunc {
+// PollingInterval is an optional function, sets a passed polling interval
+func PollingInterval(interval time.Duration) OptionFunc {
 	return func(p *PollingProjectConfigManager) {
 		p.pollingInterval = interval
 	}
 }
 
-// SetNotification is an optional function, sets a passed notification
-func SetNotification(notificationCenter notification.Center) OptionFunc {
+// Notification is an optional function, sets a passed notification
```

Comment:
nit: This should be `NotificationCenter`

Reply:
changed